org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -954,6 +954,28 @@ GithubOidcSynapseStatusLambda:
       - !Ref SynapseProdAccount
     Region: us-east-1
 
+GithubOidcBixArenaInfra:
+  Type: update-stacks
+  DependsOn: GithubOidcSageBionetworks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.7.6/templates/IAM/github-oidc-provider.j2
```

Comment:
This is a little out of date, it looks like the most recent change to the file is v0.8.9

Reply:
done